src/site/markdown/keysmap-multiple.md.vm

```diff
@@ -0,0 +1,65 @@
+Multiple KeysMpa configuration
+==============================
+
+You can use a multiple source of `keysMap` file with the plugin.
+
+Simply way
```

Comment:
Grammar: _Basic example_ or _Basic usage_ (i don't think _simply way_ is quite correct)